pkgs/development/tools/sqldef/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "sqldef";
+  version = "0.11.35";
+
+  src = fetchFromGitHub {
+    owner = "k0kubun";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-T8BhGO860G8mlvVzspR2XbHsWrVWYmsTW+tc86Bskr4=";
+  };
+
+  vendorSha256 = "sha256-f7f7VGWu8PDZhb5FlqJRfR9zPutpmN3UYuKLxvh4RDE=";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Idempotent SQL schema management tool";
+    license = licenses.mit;
```

This package has two licenses, for parser and for everything else. So license should be
```suggestion
    license = with licenses; [ mit /* for everythnig except parser */  asl20 /* for parser */ ];
```